app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -409,9 +409,9 @@ public void onFsViewerDoUiUpdate(FilesystemViewerAdapter adapter) {
                 }
 
                 @Override
-                public void onFsViewerSelected(String request, File file) {
+                public void onFsViewerSelected(String request, File file, final Integer lineNumber) {
                     if (TextFormat.isTextFile(file)) {
-                        DocumentActivity.launch(MainActivity.this, file, false, null, null);
+                        DocumentActivity.launch(MainActivity.this, file, false, null, null, lineNumber);
```

Comment:
@adelobosko  Is `lineNumber` actually used / passed in anywhere in `onFsViewerSelected`? If not, we should revert the signature and pass in null here?

Reply:
> 
> 
> @adelobosko Is `lineNumber` actually used / passed in anywhere in `onFsViewerSelected`? If not, we should revert the signature and pass in null here?

Sorry, I do not understand the question.

Reply:
> 
> 
> @adelobosko Is `lineNumber` actually used / passed in anywhere in `onFsViewerSelected`? If not, we should revert the signature and pass in null here?

if we replace lineNumber here to null then we could not move to this line in the file
![image](https://user-images.githubusercontent.com/56083544/119893234-c8f2c700-bf43-11eb-852b-d892bc7a380b.png)